tools/importer/services/getBlocks.js

```diff
@@ -1,6 +1,14 @@
-export default async function getBlocks() {
-  // update mock url later
-  const response = await fetch('https://run.mocky.io/v3/9325bf4a-5be9-4700-8b0c-ad60e25e1693');
-  const data = await response.json();
-  return data;
+export default async function getBlocks(url) {
+  try {
+    const parsedURL = new URL(url)
```

Comment:
I see a semicolon is missing for most of the lines.
Please run eslint(Already configured) before creating the PR.

Reply:
@Deva309 Please review.